🥳[Techdebt] relayout address view controller

kcw-grunt opened the pull request:
Big refactor of the Send View Controller
- Add UIHostingViewControllers for the Cells
- Fixes the covered LTC Address
- Successful Send and Receive

mosadialiou commented:
Very nice @kcw-grunt
LGTM. Just one thing, let's first merge this branch into develop.

mosadialiou reviewed (changes requested):
Very nice @kcw-grunt
LGTM. Just one thing, let's first merge this branch into develop.

kcw-grunt commented:
Switched to develop @mosadialiou

mosadialiou reviewed (changes requested):
Why all these changes? did you create the `kcw-grunt-patch-2` from `develop`? OR maybe `develop` doesn't not yet have the latest code?

kcw-grunt commented:
maybe the master changes were not brought in...gimme a sec @mosadialiou

kcw-grunt commented:
Please take a look at https://github.com/litecoin-foundation/loafwallet-ios/pull/121 first @mosadialiou

kcw-grunt commented:
Will close this until the #121 is merged.  Then I will re-open this

kcw-grunt committed:
Successfully placed in UIHosting Address

Updated the strings file

kcw-grunt committed:
Added event

kcw-grunt committed:
added delay

- used for animating U Domain

kcw-grunt committed:
Added function of subviews in SendViewController

Layout is updated

kcw-grunt committed:
Adding the function to the send

Cleaned up and wired up Cells
- UD View/Model
- Send Address View/Model
Reset the amount Label in SendViewController

kcw-grunt committed:
Refactored SendButton

- Using UIHostingViewController

kcw-grunt committed:
Successfully send LTC

-WIP: fix white space

kcw-grunt committed:
Refactored the layout of the send  button
